### ISSUE TITLE
YCB数据集过大/The YCB dataset is too large.

### ISSUE THREAD

**Issue by 123Zoey** [j96w/DenseFusion#220]
YCB数据集过大，作者可以私法给我一份吗？我的邮箱是2323521062@qq.com，谢谢！
The YCB dataset is too large. Can the author provide me with a private copy? My email is 2323521062@qq.com Thank you!

**Comment by PannagaS** [comment]
Exactly! How did you run this then? did you get a private copy? Also how do I run the inference?
